docs/configuration/keymap.md

```diff
@@ -9,288 +9,476 @@ description: Learn how to configure keyboard shortcuts with Yazi.
 If you haven't created and used your own configuration file yet, please see [Configuration](./overview.md).
 :::
 
-## [manager]
+## `[manager]`
 
-- escape: Cancel find, exit visual mode, clear selected, cancel filter, or cancel search.
+### `escape`
 
-  - `--all`: Do all of the above.
-  - `--find`: Cancel find.
-  - `--visual`: Exit visual mode.
-  - `--select`: Clear selected.
-  - `--filter`: Cancel filter.
-  - `--search`: Cancel search.
+Cancel find, exit visual mode, clear selected, cancel filter, or cancel search.
 
-  Automatically determine the operation by default, and it will only execute the selected operation after specifying the option; multiple options can be stacked.
+| Options/Arguments | Description          |
+| ----------------- | -------------------- |
+| `--all`           | Do all of the above. |
+| `--all`           | Do all of the above. |
+| `--find`          | Cancel find.         |
+| `--visual`        | Exit visual mode.    |
+| `--select`        | Clear selected.      |
+| `--filter`        | Cancel filter.       |
+| `--search`        | Cancel search.       |
 
-- quit: Exit the process.
+Automatically determine the operation by default, and it will only execute the selected operation after specifying the option; multiple options can be stacked.
 
-  - `--no-cwd-file`: Don't write the current directory to the `cwd-file`.
+### `quit`
 
-- close: Close the current tab; if it's the last tab, exit the process instead.
+Exit the process.
 
-### Navigation
+| Options/Arguments | Description                                          |
+| ----------------- | ---------------------------------------------------- |
+| `--no-cwd-file`   | Don't write the current directory to the `cwd-file`. |
 
-- arrow
+### `close`
 
-  - `n` / `n%`: Move the cursor up or down by `n` or `n%` lines. Use negative values to move up and positive values to move down.
+Close the current tab; if it's the last tab, exit the process instead.
 
-- leave: Go back to the parent directory.
-- enter: Enter the child directory.
-- back: Go back to the previous directory.
-- forward: Go forward to the next directory.
-- seek
+### `arrow`
 
-  - `n`: Seek up or down at file contents in the preview. Use negative values to peek up and positive values to peek down.
+| Options/Arguments | Description                                                                                                       |
+| ----------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `[n \| n%]`       | Move the cursor up or down by `n` or `n%` lines. Use negative values to move up and positive values to move down. |
```

Comment:
Would it be better for `[n]` / `[n%]`?

Reply:
`git --help`:

```
usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           [--config-env=<name>=<envvar>] <command> [<args>]
```

Reply:
I think `-v` and `--version` are equivalent; they have the same function and have the same behavior, but `n` and `n%` are different, the former is based on a per-line unit, while the latter is based on a percentage unit, and they have different behaviors.